services/common/src/components/reports/AddReportDetails.tsx

```diff
@@ -291,13 +297,17 @@ const AddReportDetails: FC<AddReportDetailsProps> = (props) => {
         </Col>
         <Col span={24}>
           <Form layout="vertical">
-            <Form.Item label="Report Title and Additional Comment">
-              <Input.TextArea
-                rows={3}
+            <Form.Item label="Report Title and Additional Comment*">
```

Comment:
Just to avoid some merge conflicts here- Form.Item is already within the Field element and doesn't need to be here too (I took them out, this is the only one that should make a conflict). Pass the label to Field to show it, might have to do that within a `props` object.

Reply:
Other than this one minor thing that's not much of a thing (I _can_ resolve it myself of course) everything looks good to me!

Reply:
Sounds good, the Form.Item is removed now. I ended up not using label in the field or passing it a `props `object and set up the text in a `Typography.Text` due to some styling issue, hope this is okay?